### DISCUSSION TITLE
Is the capability of handling contra-rotating rotors not very good?

### DISCUSSION THREAD

**Discussion by VE111** [byuflowlab/FLOWUnsteady#188] [Q&A]
I am currently using the rotor hover example in FLOWUnsteady to solve for the aerodynamic forces of a coaxial dual-rotor helicopter. But I find the the force of lower rotor is small. Does anyone  have the same question?

**Comment by Burning1997** [comment]
> I am currently using the rotor hover example in FLOWUnsteady to solve for the aerodynamic forces of a coaxial dual-rotor helicopter. But I find the the force of lower rotor is small. Does anyone have the same question?

Hi, I was recently looking for a way to calculate a counter rotor using flowunsteady, the tutorial only has a single rotor in it, can you share how you set up multiple rotors?

**Reply by VE111** [reply]
Hello, the method I use for calculation involves incorporating a rotor by referencing a file like "prowim.jl", and then adjusting the positions (xpos, zpos) as well as the rotation direction of the rotor.

**Reply by Burning1997** [reply]
> Hello, the method I use for calculation involves incorporating a rotor by referencing a file like "prowim.jl", and then adjusting the positions (xpos, zpos) as well as the rotation direction of the rotor.

Thank you very much for your reply, it helped me a lot. I read the prowim.jl file and it uses a wing and two identical propellers, I still don't know how to remove the wing and use two geometrically different propellers.

**Reply by VE111** [reply]
I believe it's feasible to introduce an additional 'generate_rotor' function to accommodate the use of two distinct geometric propellers.

However, I've encountered a discrepancy between my simulation results and the actual experimental data, which warrants further investigation.

I`m looking forward to your result!

**Reply by Burning1997** [reply]
I have the same problem.

**Comment by allanown** [comment]
Hello VE111,
I have the same question. It cannot predict the CT and CQ of the lower rotor. Have you solved this problem?
